sn_networking/src/event/request_response.rs

```diff
@@ -201,16 +208,13 @@ impl SwarmDriver {
             incoming_keys.len()
         );
 
-        // accept replication requests from the K_VALUE peers away,
-        // giving us some margin for replication
-        let closest_k_peers = self.get_closest_k_value_local_peers();
-        if !closest_k_peers.contains(&holder) || holder == self.self_peer_id {
-            debug!("Holder {holder:?} is self or not in replication range.");
+        let more_than_one_key = incoming_keys.len() > 1;
+        // accept replication requests from all peers within our GetRange
+        if !peers.contains(&holder) || holder == our_peer_id {
```

Comment:
`peers` is actually from `self.get_all_local_peers()` ,
which means we accept replication as long as the sender appears in our local RT ?

is it correct? or at least the comment shall be updated to : 
`accept replication requests from all peers`

Reply:
Ah is only local yes. Updated for clarity